gstreamer-sdp/src/s_d_p_message.rs

```diff
@@ -223,15 +215,15 @@ impl SDPMessage {
         }
     }
 
-    pub fn insert_attribute(&mut self, idx: i32, attr: &mut SDPAttribute) -> SDPResult {
+    pub fn insert_attribute(&mut self, idx: i32, attr: &SDPAttribute) -> SDPResult {
```

Comment:
This is taking ownership of the argument from what I can see, so should be changed to do that. After passing to native code, no drop should be called so it should probably be just forgotten with `mem::forget`